### PR TITLE
Printing warning in abs fixed

### PR DESCRIPTION
**Pull request by gvaibhavraj2**
used /| instead of /lvert

### CODE REVIEW

**Comment by debugger22** [comment]
Please mention respective issue while generating a PR. It helps the reviewer.

https://github.com/sympy/sympy/issues/8984

**Comment by smichr** [comment]
> Please mention respective issue while generating a PR. It helps the reviewer.

And since you are actually fixing the issue, if you follow the syntax given below, the issue will be closed when this is merged:
:eyeglasses: See also: **Writing pull request title and description** at [the wiki](https://github.com/sympy/sympy/wiki/Development-workflow)

**Comment by smichr** [comment]
:repeat: this will need to be merged with an updated master which has changed (#8985 addressed some of what you addressed here) and made this unmergable :eyeglasses: See **Synchronization with master sympy/sympy** at [the wiki](https://github.com/sympy/sympy/wiki/Development-workflow)

**Comment by gvaibhavraj2** [comment]
Do i need to put the issue number like "issue-8984" in the PR title?

On 18 February 2015 at 21:46, Christopher Smith <notifications@github.com>
wrote:

> [image: :repeat:] this will need to be merged with an updated master
> which has changed (#8985 <https://github.com/sympy/sympy/pull/8985>
> addressed some of what you addressed here) and made this unmergable [image:
> :eyeglasses:] See *Synchronization with master sympy/sympy* at the wiki
> <https://github.com/sympy/sympy/wiki/Development-workflow>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/sympy/pull/9027#issuecomment-74892771>.
>



-- 
Vaibhav Raj Gupta
Sophomore,
Computer Science And Engineering ,
IIT ROORKEE,INDIA

**Comment by debugger22** [comment]
> Do i need to put the issue number like "issue-8984" in the PR title?

No. See [this](https://help.github.com/articles/closing-issues-via-commit-messages/).

**Comment by smichr** [comment]
Anyone else please feel free to commit when this is ready. I think the changes here are good since they remove the `\lvert` which apparently isn't being tested but if it were would be generating the same problems as using `\lvert` with Abs was causing (as described in #8984).

File: sympy/printing/latex.py [hunk 1]
```diff
@@ -87,10 +87,10 @@
     'scr': lambda s: r'\mathscr{'+s+r'}',
     'frak': lambda s: r'\mathfrak{'+s+r'}',
     # Brackets
-    'norm': lambda s: r'\left\lVert{'+s+r'}\right\rVert',
+    'norm': lambda s: r'\left\|{'+s+r'}\right\|',
     'avg': lambda s: r'\left\langle{'+s+r'}\right\rangle',
-    'abs': lambda s: r'\left\lvert{'+s+r'}\right\rvert',
-    'mag': lambda s: r'\left\lvert{'+s+r'}\right\rvert',
+    'abs': lambda s: r'\left\|{'+s+r'}\right\|',
+    'mag': lambda s: r'\left\|{'+s+r'}\right\|',
```
Comment (cbm755) [inline sympy/printing/latex.py]: these too look wrong, isn't `\|` two bars?   like a norm?  You can't replace both `\lVert` and `\lvert` with the same thing!

File: sympy/printing/latex.py [hunk 2]
```diff
@@ -87,10 +87,10 @@
     'scr': lambda s: r'\mathscr{'+s+r'}',
     'frak': lambda s: r'\mathfrak{'+s+r'}',
     # Brackets
-    'norm': lambda s: r'\left\lVert{'+s+r'}\right\rVert',
+    'norm': lambda s: r'\left\|{'+s+r'}\right\|',
```
Comment (smichr) [inline sympy/printing/latex.py]: see #9039 -- should this be `\left|`?

Comment (cbm755) [inline sympy/printing/latex.py]: No it should not.  `\lVert` and `\` give two bars.  (`\lvert` and `|` give one.)

**Comment by smichr** [comment]
OK, `abs` was handled in #9039. 

In this PR, `abs` should be left alone, `mag` should be fixed and `norm` is corrected correctly. So I think the only things to be done are

- [ ] undo changes to `abs`
- [ ] fix mag
- [ ] add tests for mag and norm

**Pull request closed without merging**